sass/themes/schemas/components/dark/_toast.scss

```diff
@@ -9,40 +9,16 @@
 ////
 
 /// Generates a dark base toast schema.
-/// @type {Map}
-/// @prop {Map} background [color: ('gray', 500, .9)] - The background color used for the toast.
-/// @prop {Map} border-color [color: ('gray', 500, .9)] - The border color used for the toast.
-/// @prop {Map} text-color [contrast-color: ('gray', 500)] - The text-color used for the toast.
-$dark-base-toast: (
-    background: (
-        color: (
-            'gray',
-            500,
-            0.9,
-        ),
-    ),
-    border-color: (
-        color: (
-            'gray',
-            500,
-            0.9,
-        ),
-    ),
-    text-color: (
-        contrast-color: (
-            'gray',
-            500,
-        ),
-    ),
-);
+/// @see $light-toast
+/// @see $bootstrap-toast
+$dark-base-toast: $light-toast;
```

Comment:
The $dark-base-toast is not used anywhere, so we may want to remove it.

Reply:
I will retain it for consistency.